ui/library/less/inputs.less

```diff
@@ -0,0 +1,12 @@
+// Hide the spinner in the page number control
+/* Chrome, Safari, Edge, Opera */
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
```

Comment:
Doing this globally would be a _very_ bad idea. You are effectively making changes to elements outside the library.